pkg/storage/secret/authorizer.go

```diff
@@ -10,6 +10,7 @@ type authorizer struct {
 	// TODO!!!
 }
 
+// TODO: why not `CanCreate`?
```

Comment:
👍  Both for create and delete, not sure what `On` means in this context, what would be applied or expected to happen